config/initializers/wrap_parameters.rb

```diff
@@ -5,7 +5,8 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  #wrap_parameters format: [:json]
```

Comment:
Don't leave commented lines (unless it comes with something like `TODO: revert to this once X thing is done`)